src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+html {
+  font-family: "Roboto", "Helvetica", sans-serif;
+  color: #060b35;
+  margin: 0;
+  --secondary-color: #616070 ;
+  --blue-color: #00acdc;
+  --white-color: #fff;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  height: 408px;
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border: 1px #f3f3f3;
```

Comment:
why don't you use parent selector?
https://sass-lang.com/documentation/style-rules/parent-selector/